src/reflect/scala/reflect/api/Trees.scala

```diff
@@ -2187,7 +2187,7 @@ trait Trees { self: Universe =>
   /** 0-1 argument list new, based on a symbol.
    *  @group Factories
    */
-  @deprecated("use q\"new ${sym.toType}(..$args)\" instead", "2.10.1")
+  @deprecated(s"""use q"new $${sym.toType}(..$$args)" instead""", "2.10.1")
```

Comment:
Does this work? Color me surprised.

Reply:
Me too. I tried just escaping the unescapable dollar, but no dice.

Reply:
My oneliner to make this work is still under SIP: https://github.com/scala/docs.scala-lang/blob/master/_sips/sips/2018-07-31-interpolation-quote-escape.md

It's likely the SIP committee will get around to it in the first or second SIP meeting after the scala 3 SIPs have gone through the pipeline.